### PR TITLE
Add gelidum

### PR DESCRIPTION
**Pull request by johnomotani**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by johnomotani** [comment]
@diegojromerolopez could you please comment here to confirm you're happy to be a maintainer for this feedstock?

**Comment by diegojromerolopez** [comment]
Yeah, I'm happy @johnomotani.

**Comment by johnomotani** [comment]
Thanks @diegojromerolopez!

@conda-forge/help-python I think this PR is ready for review merge. Thanks!

File: recipes/gelidum/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  sha256: 67ddde79967a39799414009710edc0b306e5b210e7ec835739f5172eb0498f31
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```
Comment (dopplershift) [inline recipes/gelidum/meta.yaml:19]: Need a version requirement here:
```suggestion
    - python >=3.7
```

File: recipes/gelidum/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - pip
+  run:
+    - python >=3.7.0
```
Comment (dopplershift) [inline recipes/gelidum/meta.yaml:22]: Since we need to change above, we usually just use:
```suggestion
    - python >=3.7
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Recipe is in great shape! Just a couple minor nits to fix and this can go in.

**Comment by johnomotani** [comment]
Thanks @dopplershift, fixed now!

**Review by dopplershift — APPROVED** [review]

**Pull request merged**